estate/views/estate_property_tag_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0'?>
+<odoo>
+    <record id="estate_property_tag_views" model="ir.actions.act_window">
```

Comment:
```suggestion
    <record id="action_estate_property_tag" model="ir.actions.act_window">
```
Action name must always start with `action_` followed by what it does. Please ensure it for all actions implemented